python/tk_multi_loader/dialog.py

```diff
@@ -1418,9 +1418,20 @@ def _setup_hierarchy_model(self, app, root):
         :return: Created `(model, proxy model)`.
         """
 
+        # If the root is a aproject, include it in the hierarchy model so that
+        # we can display project publishes. We do an innocent little hack here
+        # by including a space at the front of the project root item to make it
+        # display first in the tree.
+        include_root = " Project Publishes" if root.get("type") == "Project" else None
```

Comment:
I would prefer if you didn't do this hack and instead derived a proxy model from QSortFilterproxyModel and reimplement [lessThan](http://doc.qt.io/qt-4.8/qsortfilterproxymodel.html#lessThan).